packages/react/src/components/Carousel/Carousel.tsx

```diff
@@ -278,6 +278,7 @@ class Carousel extends AutoControlledComponent<WithAsProp<CarouselProps>, Carous
                     defaultProps: () => ({
                       active: activeIndex === index,
                       id: itemIds[index],
+                      navigation: !!this.props.navigation,
```

Comment:
probably the cast is not required, can be undefined.

Reply:
this is good point :) probably is not needed, but it looks for me simple way.. :)

I though sent there just boolean value, instead of the shorthand values/collection. 
As in the carouselItemBehavior is boolean efficient. 
I think if navigation prop would be undefined, then it should be fine, it will result as false (as I tried run it).    

So I am not sure now... :)

Reply:
Carousel can have navigation undefined, there is no reason to make it a boolean here.

It won't make much of a difference anyway, do as you like.